rozenborg/test-simulations-mgmt-repo
# Chores(Outputs: Release Note, Peer Review, Solving the problem, Design Patterns, HTML templates, Setting up software development environments)

#### What does this PR do?
- Fulfills the following outputs.
1. Delivering your Product (Release Notes). 
2. Getting Peer reviews.
3. Solving the problem.
4. Design patterns used.
5. Writing Front-End template.
6. Setup software development environment.

#### Description of Task to be completed?
- Create a release notes for your final product in your repo's wiki
- Get at least two peer code reviews
- Implement the solution.
- Explain design patterns used.
- Implement a template for the project's front-end.
- Setup production and staging app on Heroku.
- Link github repo to Heroku


#### How should this be manually tested?
- Clicking on the project `Wiki` tab should bring up the release notes
- Pull request link includes the peer code review
- View the file `InvertedIndex.js` to check for code implementation.
- Visit [app link](https://inverdex-staging.herokuapp.com/)
